docker-compose.yml

```diff
@@ -18,4 +18,6 @@ services:
         working_dir: /project/source
         command: >
           bash -c "/opt/conda/bin/pip3 install -r /project/requirements.txt
-          && /opt/conda/bin/python3 main.py client"
\ No newline at end of file
+          && /opt/conda/bin/python3 main.py client"
+        ports:
+            - "5005:5000"
```

Comment:
De ce e nevoie de 5005 aici ?

Reply:
Pentru ca a expus in plus de clientul MQTT un server FastAPI din client. TBH nu sunt sigur la ce foloseste serverul, dar e nevoie sa expuna public portul ca sa il poata vedea dinafara.